### PR TITLE
Add radiation notebook

### PR DESCRIPTION
**Pull request by MackyDIARRA**

### CODE REVIEW

File: examples/radiation.ipynb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (martinRenou) [inline examples/radiation.ipynb:1]: Can you rename the Notebook with a capital letter "Radiation"? The other notebooks have capital letters.

**Review by martinRenou — CHANGES_REQUESTED** [review]
Thanks!

File: examples/radiation.ipynb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'Ipyleaflet'"
```
Comment (davidbrochart) [inline examples/radiation.ipynb:17]: Maybe clear the outputs?

File: examples/CountriesGeoJSON.ipynb [hunk 3]
```diff
@@ -60,9 +82,9 @@
...
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.9.7"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 4
 }
```
Comment (davidbrochart) [inline examples/CountriesGeoJSON.ipynb:90]: I think you didn't mean to commit this file.

File: examples/radiation.ipynb [hunk 4]
```diff
@@ -2,20 +2,23 @@
...
+       "model_id": "e6e1117099a34472927b621c0c6b7e70",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Map(center=[47, 2], controls=(ZoomControl(options=['position', 'zoom_in_text', 'zoom_in_title', 'zoom_out_text…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
```
Comment (davidbrochart) [inline examples/radiation.ipynb:21]: You can use "Clear all outputs" in Jupyter Notebook or JupyterLab, and save the notebook. That way, the notebook will only have the source code.

**Pull request merged**